src/containers/support/SupportFAQ.tsx

```diff
@@ -157,7 +157,7 @@ const supportQuestions: SupportQuestion[] = [
 
 const headerProps = {
   description:
-    "We've compiled a list of FAQs for the VA API program; our goal is to get your question answered as soon as possible. If you need further help\u2014or have comments or feedback\u2014please contact us via GitHub or use our short 'Contact Us' form. Our customer support team will be happy to help.",
+    "We've compiled a list of FAQs for the VA API program; our goal is to get your question answered as soon as possible. If you need further help\u2014or have comments or feedback\u2014please contact us by using our short 'Contact Us' form. Our customer support team will be happy to help.",
```

Comment:
I don't know if there's a reason for using dashes here instead of commas because it's a piece of header text but I've always seen this style of aside in a sentence using commas to break it out.

```suggestion
    "We've compiled a list of FAQs for the VA API program; our goal is to get your question answered as soon as possible. If you need further help, or have comments or feedback, please contact us by using our short 'Contact Us' form. Our customer support team will be happy to help.",
```